readme.md

```diff
@@ -3,15 +3,15 @@
 # Skipper
 
 Skipper is an HTTP router that acts as a reverse proxy (with support for flexible route definitions), alters
-requests, and responses with filters. It:
+requests, and responds with filters. It:
```

Comment:
"request -> response" is the way the http people refer to requests responds :) you shouldn't change this...

https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Response_message